src/play_sound_event.rs

```diff
@@ -1,5 +1,15 @@
-use bevy::prelude::Event;
+use libfmod::Studio;
 
-// TODO: Replace this with an EventDescription?
-#[derive(Event)]
-pub struct PlaySoundEvent(pub &'static str);
+pub struct OneShotPlayer {
+    pub studio: Studio,
+}
+
+impl OneShotPlayer {
+    pub fn play_event(&self, event: &'static str) {
+        let event_description = self.studio.get_event(event).unwrap();
+        let instance = event_description.create_instance().unwrap();
+
+        instance.start().unwrap();
+        instance.release().unwrap();
```

Comment:
Haven't discussed this yet, but I'd like to avoid the use of `unwrap`, especially since this project is a library. 

I prefer to use the `anyhow` crate with `bevy_mod_sysfail` whenever possible. In some cases, you'll have to resort to `expect`. It's always valuable for developers to know what went wrong with a descriptive error message.

Reply:
I couldn't get it to work with sysfail here so I went with `expect`. Other unwraps have been removed as well